addons/survey/views/survey_templates_statistics.xml

```diff
@@ -145,6 +110,57 @@
         </div>
     </template>
 
+    <template id="survey_results_filters" name="Survey: Filter results">
+        <t t-set="search_passed_or_failed" t-value="search_failed or search_passed"/>
+        <div class="card-header">
+            <span class="fa fa-filter"></span>  Filters
+            <span t-if="search_filters or search_finished or search_passed_or_failed" class="float-right clear_survey_filter text-primary"><i class="fa fa-times"></i> Clear All Filters</span>
+        </div>
+        <div class="card-body question_and_page_data o_survey_result p-0">
+            <nav class="navbar navbar-light rounded">
+                <div t-if="question_and_page_data" class="d-flex flex-column flex-sm-row justify-content-between w-100">
+                    <ul class="o_survey_results_topbar_filters nav">
+                        <t t-set="dropdown_item_classes">dropdown-item d-flex align-items-center justify-content-between</t>
+                        <li class="nav-item dropdown mr-2 my-1">
+                            <a href="#" role="button" t-attf-class="btn btn-outline-primary dropdown-toggle #{'active' if search_finished else ''}" data-toggle="dropdown">
+                                <t t-esc="'All surveys' if not search_finished else 'Completed surveys'"/>
```

Comment:
A bit tricky but this will not be handled correctly for translations.
As we extract tag content, you need to do rather:

```xml
<span t-if="search_finished">Completed survey</span>
<span t-else="">All survey</span>
```